src/components/elements/select/select.njk

```diff
@@ -1,8 +1,23 @@
-<p>
+{% set ariaDescribedby = '' %}
+{% if hint %}
+    {% set ariaDescribedby = name + '-hint' %}
+{% elif error %}
+    {% set ariaDescribedby = name + '-error' %}
+{% elif hint and error %}
+    {% set ariaDescribedby = name + '-hint' + " " + name + '-error' %}
+{% endif %}
+
+<div class="field{% if error %} field--error{% endif %}">
     <label for="{{ name }}">{{ label }}</label>
     <select id="{{ id if id else name }}" name="{{ name }}">
         {%- for value, label in options -%}
         <option value="{{ value }}"{% if default == value %} selected{% endif %}>{{ label }}</option>
         {%- endfor -%}
     </select>
```

Comment:
I believe the `ariaDescribedby` is missing from the `<select>`

Reply:
Fixed in d9e4f3e.